ecosystem/sep-0012.md

```diff
@@ -6,7 +6,7 @@ Title: KYC API
 Author: Interstellar
 Status: Active
 Created: 2018-09-11
-Updated: 2022-07-07
+Updated: 2022-08-29
 Version 1.9.1
```

Comment:
Lets bump the patch version to 1.9.2

Reply:
Addressed in https://github.com/stellar/stellar-protocol/pull/1302/commits/fd40428111ccd66e77f36ebefcbdaf087d2e9ea8.